### ISSUE TITLE
tsearch: Point not inserted

### ISSUE THREAD

**Issue by Jean-Romain** [davidcsterratt/geometry#46]
Reproducible example given by email:

``` r
library(geometry)
source("dumpdata.txt")
D = delaunayn(cbind(x,y))
tsearch(x,y,D,x1,y1)
#> Error in C_tsearch(x, y, t, xi, yi, bary): Failed to insert point into QuadTree.
#> Please post input to tsearch  (or tsearchn at
#> https://github.com/davidcsterratt/geometry/issues
#> or email the maintainer.
```
:bookmark: [dumpdata.txt](https://github.com/davidcsterratt/geometry/files/3768760/dumpdata.txt)

**Comment by Jean-Romain** [comment]
As expected it is a problem of computer precision. At the level 6 the quadrants are no longer aligned with the main bounding box of the quadtree because of the successive division by 2. There is nothing to do but increasing the default epsilon to `1e-10`. I think epsilon should be a parameter modifiable by the user.

https://github.com/davidcsterratt/geometry/blob/192fc01ea0d4ba741cafe24e2ae1d411520ea69c/pkg/src/Rtsearch.cpp#L59

**Comment by cbec** [comment]
Greetings,  

I have run into this same issue ("Failed to insert point into QuadTree."), and have tried to edit Rtsearch.cpp function as stated above and reinstall the package to no avail.  If I use the "orig" method in place of the "quadtree" method the function appears to work very well.  I will try and work up a smaller example dataset to share.  The code as it stands would need some simplification to share.  

Thanks for all your work on this!

**Comment by davidcsterratt** [comment]
Seems to have been fixed by #57 , now in the master branch.
